contracts/stargaze-mutable-whitelist/src/contract.rs

```diff
@@ -37,14 +37,20 @@ pub fn execute(
     msg: ExecuteMsg,
 ) -> Result<Response, ContractError> {
     match msg {
-        ExecuteMsg::AddAddress { address } => execute_add_address(deps, address),
-        ExecuteMsg::RemoveAddress { address } => execute_remove_address(deps, address),
+        ExecuteMsg::AddAddress { address } => execute_add_address(deps, info, address),
+        ExecuteMsg::RemoveAddress { address } => execute_remove_address(deps, info, address),
         ExecuteMsg::UpdateOwnership(action) => update_ownership(deps, env, info, action),
```

Comment:
I guess you need to assert_owner in update_ownership as well?

nit: could also retitle to `execute_update_ownership`